_ml-commons-plugin/model-access-control.md

```diff
@@ -114,4 +114,65 @@ PUT _cluster/settings
 
 Model access control is achieved through the Model Group APIs. These APIs include the register, search, update, and delete model group operations.
 
-For information about model access control API, see [Model group APIs]({{site.url}}{{site.baseurl}}/ml-commons-plugin/api/model-group-apis/index/).
\ No newline at end of file
+For information about model access control API, see [Model group APIs]({{site.url}}{{site.baseurl}}/ml-commons-plugin/api/model-group-apis/index/).
```

Comment:
Should the API name be capped, i.e., "Model Access Control API?"

Reply:
Yes, thanks!